components/post_emoji/post_emoji.test.tsx

```diff
@@ -20,6 +20,12 @@ describe('PostEmoji', () => {
         });
     });
 
+    test('should render shortcode text within span when imageUrl is provided', () => {
```

Comment:
I'd like to add e2e testing, but it appears native browser commands aren't really supported by Cypress. Asking around, but it's looking like we'll have to add copy/paste testing to our manual suite of test.

Reply:
Perhaps @jgilliam17 can shed some more light. I'll tag @saturninoabril here as well - maybe he has some insights to share.

Reply:
@bradjcoughlin re: e2e, maybe just checking the text contents (`cy.contains`?) of a post with emoji is good enough? (leaving the selection+copy/paste flows for another test to handle)

Reply:
@calebroseland yeah, I thought about that as well. I can add that to the existing e2e test, but of course it doesn't quite mimic the real-world interaction. But I agree it's better than nothing.

Reply:
I added a couple Cypress tests just to verify the element content as you suggested 👍

Reply:
+1 to @calebroseland's suggestion. Though it's a Cypress limitation, copy/paste starts with a text to select/highlight and so some test like below is good enough for minimum test requirement.
```javascript
const message = 'This post has a shortcode emoji :raising_hand_man: within it.';
cy.postMessage(message);

cy.getLastPostId().then((postId) => {
    cy.get(`#postMessageText_${postId}`).should('have.text', message);
});
```